src/utils.ts

```diff
@@ -123,7 +123,7 @@ export const generateGradientColors = () => {
   return [
     [h, 50, b],
     [h, 70, b - 14],
-    [h, 62, b - (dark ? -16 : 20)],
+    [h, 62, b - (dark ? -10 : 16)],
```

Comment:
for that these fix?

Reply:
the task said that when changing the style of the card to a gradient, there was a solid color, but in fact it is not, I tried to increase the contrast of the gradient by quite a bit this way, to make it visible

Reply:
these indexes must not be touched, they have been approved by the designer